### PR TITLE
rpc: add documentation for genesis chunked api

### PR DESCRIPTION
**Pull request by tychoish**
closes: #6721

### CODE REVIEW

File: rpc/openapi/openapi.yaml [hunk 1]
```diff
@@ -808,12 +808,43 @@ paths:
                 $ref: "#/components/schemas/ErrorResponse"
   /genesis:
     get:
-      summary: Get Genesis
+      summary: Get Genesis in chunked
```
Comment (marbar3778) [inline rpc/openapi/openapi.yaml:811]:
```suggestion
      summary: Get Genesis
```

/genesis and /genesis_chunked are two different routes. 

https://github.com/tendermint/tendermint/blob/master/rpc/core/routes.go#L24

**Review by marbar3778 — COMMENTED** [review]
thanks for taking this on!!

File: rpc/openapi/openapi.yaml (hunk 1 above)
Comment (tychoish) [reply rpc/openapi/openapi.yaml:811]: Eep. Sorry about this!

Comment (tychoish) [reply rpc/openapi/openapi.yaml]: Eep. Sorry about this!

File: rpc/openapi/openapi.yaml [hunk 2]
```diff
@@ -813,7 +813,38 @@ paths:
...
+              schema:
+                $ref: "#/components/schemas/GenesisChunkedResponse"
+        "500":
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+
+  /genesis_chunked:
```
Comment (marbar3778) [inline rpc/openapi/openapi.yaml:840]:
```suggestion
  /genesis
```

then the route above would be _chunked?

**Review by marbar3778 — APPROVED** [review]
Awesomesauce!!

**Pull request merged**